pkg/fd/fd.go

```diff
@@ -185,6 +186,12 @@ func OpenAt(dir *FD, path string, flags int, mode uint32) (*FD, error) {
 	return New(f), nil
 }
 
+// OpenUnix Open a Unix Domain Socket and return the file descriptor for it.
```

Comment:
You can't open a unix socket. You can connect to it though.

`// OpenUnix connects to a Unix Domain Socket...`